script/draft-release/run.ts

```diff
@@ -96,21 +111,81 @@ export async function run(args: ReadonlyArray<string>): Promise<void> {
   const noChangesFound = lines.every(l => l.trim().length === 0)
 
   if (noChangesFound) {
+    console.warn('No new changes found to add to the changelog. Aborting.')
     // print instructions with no changelog included
     printInstructions(nextVersion, [])
-  } else {
-    const changelogEntries = await convertToChangelogFormat(lines)
-
-    console.log("Here's what you should do next:\n")
-
-    if (channel === 'production') {
-      // make sure we only include entries since the latest production release
-      const existingChangelog = getChangelogEntriesSince(previousVersion)
-      const entries = [...existingChangelog]
-      printInstructions(nextVersion, entries)
-    } else if (channel === 'beta') {
-      const entries = [...changelogEntries]
-      printInstructions(nextVersion, entries)
+    return
```

Comment:
Question: wondering if we want to wait to return out of the function until after we update the app version?

Reply:
mmm good point! i restructured things for this to work better in a39c3cfe101279d75595ac180d868a12db64605b